startup.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+
+export ACCESS_LOG_FORMAT='%(h)s %(l)s %({REMOTE_USER}i)s %(t)s "%(r)s" "%(q)s" %(s)s %(b)s %(D)s'
+
+source /cds/group/pcds/pyps/conda/venvs/lcls_naming_tool/bin/activate
+
+cp ./lcls_naming_tool/lcls_naming_tool.py ./lcls_naming_tool/web
+
+cp -r ./lcls_naming_tool/taxons ./lcls_naming_tool/web
```

Comment:
Possibly there's a way to rethink this so that we don't need to copy files around at startup

Reply:
These lines are for the flask app.py to work. I tried a few workarounds but didn't find a good solution, so ended up copying the files. Will find a better solution in the next PR.

Reply:
Looks like `LCLS_NAMING_TOOL_TAXONS_DIR` will be defined for the CLI tool but not sure how it will be set for the pswww server. I've added `export LCLS_NAMING_TOOL_TAXONS_DIR="$(pwd)/lcls_naming_tool/taxons"` in startup.sh for the web server for now.